examples/samg55_xplained_pro/spi-loopback/main.cpp

```diff
@@ -0,0 +1,37 @@
+#include "modm/platform.hpp"
+#include "modm/board.hpp"
```

Comment:
```suggestion
#include <modm/board.hpp>
```

No need for modm/platform.hpp since modm/board.hpp pulls that in.